tempesta_fw/vhost.h

```diff
@@ -60,6 +61,12 @@ typedef struct {
 	TfwCaPolicy	**capo;
 } TfwLocation;
 
+/* Cache purge configuration modes. */
+enum {
+	TFW_D_CACHE_PURGE_INVALIDATE,
+	TFW_D_CACHE_PURGE_PURGE,
```

Comment:
Probably `TFW_D_CACHE_PURGE_DELETE` is more clear than `TFW_D_CACHE_PURGE_PURGE`.